index.html

```diff
@@ -0,0 +1,289 @@
+							<li class="Job-title_and_year">
+								Back End Dev
+							</li>
+							<li class="Job-title_and_year">
+								2023
+							</li>
+						</ul>
+					</div>
+					<p class="notice font-style">
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<ul class="programming-languages font-style">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<button id="2" class="project-button button" type="button">See Project</button>
+				</div>
+			</div>
+			<div class="sec-div card fourth-card">
+				<img class="snapshoot-style" src="images/Snapshoot Portfolio4.svg" alt="Snapshoot portfolio image">
+				<div class="sec-div">
+					<h1 class="card-header font-style on-small">
+						Multi-Post <br> Stories
+					</h1>
+					<div class="sec-div card-description">
+						<ul class="font-style">
+							<li class="card-list-title on-small"> CANOPY</li>
+							<li class="Job-title_and_year on-small">
+								Back End Dev
+							</li>
+							<li class="Job-title_and_year">
+								2023
+							</li>
+						</ul>
+					</div>
+					<p class="notice font-style">
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<ul class="programming-languages font-style">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<button id="3" class="project-button button" type="button">See Project</button>
+				</div>
+		</section>
+	</div>
+	</div>
+	<div id="about" class="about-me">
+		<section class="written-info">
+			<section>
+				<h2 class="about-me-heading font-style">About <br> Myself</h2>
+				<p class="description font-style">
+					I’m a software developer! I can help you build a product , feature or
+					website Look through some of my work and experience! If you like what
+					you see and have a project you need coded, don’t hestiate to contact
+					me.
+				</p>
+			</section>
+
+			<section>
+				<h4 class="connect font-style">LET'S CONNECT</h4>
+				<ul class="socials">
+					<li>
+						<a class="social_medias twitter" href="https://twitter.com/FentahunAgmas">
+							<img src="images/Vector (1).svg" alt="twitter logo">
+						</a>
+					</li>
+
+					<li>
+						<a class="social_medias" href="https://www.linkedin.com/in/fentahun-agmas-5a01aa162/">
+							<img src="images/Linkedin icon.svg" alt="Linkedin logo">
+						</a>
+					</li>
+					<li>
+						<a class="social_medias" href="">
+							<img src="images/mediumLogo.png" alt="Medium icon" class="medium" />
+						</a>
+					</li>
+					<li>
+						<a class="social_medias" href="https://github.com/Fentahunn">
+							<img src="images/githublogo.png" alt="Github icon" />
+						</a>
+					</li>
+					<li>
+						<a href="#"><img src="images/Vector (2).svg" alt="two fingers"></a>
+					</li>
+				</ul>
+				<button type="button" class="resume-button button">Get my resume</button>
+			</section>

```

Comment:
As mentioned above, kindly, I suggest you use a `div` tag here instead of a `section` tag to follow best practices. 😄